pip/poetry/poetry_unit_test.go

```diff
@@ -0,0 +1,68 @@
+package poetry
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const testDataPath = "testdata"
+
+func setupPoetry(path string) *Poetry {
+	poetry := New()
+	err := poetry.SetRootModule(path)
+	if err != nil {
+		panic(err)
+	}
```

Comment:
Pass here the testing context so that everything gets wired together

```suggestion
func setupPoetry(t *testing.T, path string) *Poetry {
	poetry := New()
	err := poetry.SetRootModule(path)
	require.NoeError(t, err)
```

Reply:
Resolved via https://github.com/opensbom-generator/parsers/pull/63/commits/1bc6e9accf70a33a7458bcaa227485d0d5535652